src/main/resources/french.yml

```diff
@@ -1,5 +1,5 @@
 name: SiegeWar
-version: 0.10
+version: 0.12
 language: french
```

Comment:
Same here, could probably go to 0.11 instead of 0.12.

Reply:
Fixed